app/src/pages/HomePage.vue

```diff
@@ -5,10 +5,14 @@ import IgnorePagePadding from "@/components/IgnorePagePadding.vue";
 import { useAuth0 } from "@auth0/auth0-vue";
 import { DocType, TagType, db } from "luminary-shared";
 import { appLanguageIdAsRef } from "@/globalConfig";
+import { waitForDataRender } from "@/util/waitForDataRender";
 
 const { isAuthenticated } = useAuth0();
+console.log(isAuthenticated.value);
 
 const hasPosts = db.someByTypeAsRef(DocType.Post);
+
+const waitForDataRenderIsOver = waitForDataRender(4.5);
```

Comment:
Some comments:
The name is not really describing what we are waiting for? (It takes a moment for the data to be loaded from the database before homepage components can render the data.) Perhaps a better name would be```noContentMessageDelay```?

On slow devices it can take some time for the query to load, but 4.5 seconds is perhaps a bit much. You also want to show something before the user thinks the app is broken. Perhaps 1 second could work?

Implementing the delay as a separate (imported) function looks nice and clean, but as this is such a small thing, could it rather have been done with e.g. a ref and a timer setting the value? Perhaps something like this?
```suggestion
const noContentMessageDelay = ref(false);
setTimeout(() => { noContentMessageDelay.value = true }, 1000);
```